### DISCUSSION TITLE
How to changes the second material of model?

### DISCUSSION THREAD

**Discussion by salii15** [google/model-viewer#3917] [Q&A] [answered]
Hi, I created the material color changer but its working on only first material. How can I change the other materials colors? Thanks

![image](https://user-images.githubusercontent.com/68233176/199206568-04e0c99e-3733-43cc-b512-a6509ab7f011.png)

**Comment by WadeMorrison** [comment, marked as answer]
Hello! If I understand correctly, when you do: 
```const [material] = modelViewerColor.model.materials;``` 
it will default to the [0] color. 

Depending on how many and where the colors you want to change you just simply add [1] to the end like so:
```const [material] = modelViewerColor.model.materials[0];```
```const [material1] = modelViewerColor.model.materials[1];```

Hope this helps!

**Reply by salii15** [reply]
yes, its working. thank you.
